common/src/java/com/zimbra/common/soap/MailConstants.java

```diff
@@ -1339,4 +1339,12 @@ private MailConstants() {
     public static final String OP_INHERIT = "inherit";
     public static final String OP_MUTE = "mute";
     public static final String OP_RESET_IMAP_UID = "resetimapuid";
+
+    // Contacts API
+    public static final String E_RESTORE_CONTACTS_REQUEST = "RestoreContactsRequest";
+    public static final String E_RESTORE_CONTACTS_RESPONSE = "RestoreContactsResponse";
+    public static final QName RESTORE_CONTACTS_REQUEST = QName.get(E_RESTORE_CONTACTS_REQUEST, NAMESPACE);
+    public static final QName RESTORE_CONTACTS_RESPONSE = QName.get(E_RESTORE_CONTACTS_RESPONSE, NAMESPACE);
+    public static final String A_CONTACTS_BAKCUP_FILE_NAME = "contactsBackupFileName";
```

Comment:
Typo in this and the next String?  s/BAKCUP/BACKUP/

Reply:
Fixed